sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -600,11 +600,11 @@ case class PartitionStatistics(numFiles: Int, totalSize: Long)
  *   MSCK REPAIR TABLE table [{ADD|DROP|SYNC} PARTITIONS];
  * }}}
  */
-case class AlterTableRecoverPartitionsCommand(
+case class RepairTableCommand(
```

Comment:
I'm a bit reluctant to rename v1 commands, as they are legacy and have been there for many years.

Reply:
Sure, but this is non-public API. We have already extended `AlterTableRecoverPartitionsCommand` in https://github.com/apache/spark/pull/31499. After that, it become incompatible by sources and binary with the previous version.

Reply:
And how about the logical nodes that were massively replaced in https://issues.apache.org/jira/browse/SPARK-29900. They are legacy and have been there for many years too, aren't they?

Reply:
I think the point is that we might better focus on checking V2 instead of V1 commands because eventually V2 commands should replace V1 commands in the future. If this is likely only the one, I think it's fine. If there are a lot of such instances, I wouldn't change for now.

Reply:
> If this is likely only the one, I think it's fine.

Yep, only this for now.